tests/utils.go

```diff
@@ -1533,23 +1533,21 @@ func RunCommandOnVmiTargetPod(vmi *v1.VirtualMachineInstance, command []string)
 func NewRandomVirtualMachine(vmi *v1.VirtualMachineInstance, running bool) *v1.VirtualMachine {
 	name := vmi.Name
 	namespace := vmi.Namespace
-	vmLabels := map[string]string{"name": name}
```

Comment:
These changes likely broke an existing test, can you please investigate?

Reply:
yes, finish